src/metabase/email/password_reset.mustache

```diff
@@ -5,11 +5,18 @@
       <a href="{{hostname}}">Go to {{applicationName}}</a>
     {{/sso}}
     {{^sso}}
-    <div style="text-align: center">
-      <p>Click the button below to reset the password for your {{applicationName}} account at {{hostname}}.</p>
-      <a style="display: inline-block; box-sizing: border-box; font-size: 18px; padding: 8px 22px; cursor: pointer; text-decoration: none; border-radius: 4px; background-color: #4990E2; border-color: #4990E2; color: #fff;" href="{{passwordResetUrl}}">Reset password</a>
-      <p style="padding-top: 2em; font-size: small;">Didn't request this password reset? It's safe to ignore it.</p>
-    </div>
+      {{#isActive}}
+        <div style="text-align: center">
+          <p>Click the button below to reset the password for your {{applicationName}} account at {{hostname}}.</p>
+          <a style="display: inline-block; box-sizing: border-box; font-size: 18px; padding: 8px 22px; cursor: pointer; text-decoration: none; border-radius: 4px; background-color: #4990E2; border-color: #4990E2; color: #fff;" href="{{passwordResetUrl}}">Reset password</a>
+          <p style="padding-top: 2em; font-size: small;">Didn't request this password reset? It's safe to ignore it.</p>
+        </div>
+      {{/isActive}}
+      {{^isActive}}
+        <p>Someone requested a password reset for your {{applicationName}} account at {{hostname}}, but your account
+        has been deactivated. Contact an administrator for further assistance.</p>
+        <a href="{{hostname}}">Go to {{applicationName}}</a>
```

Comment:
I think a more sensical call-to-action would be `<a href="mailto:{{adminEmail}}">Contact your administrator</a>` — if your account isn't active, you probably don't need to go to your Metabase right now. 

Note that I'm 0% sure that `adminEmail` is in fact the name of that variable, but I'm talking about this email that we capture in `/admin/settings/general`:

![image](https://user-images.githubusercontent.com/2223916/123715809-f0cc9780-d82d-11eb-9727-6cbc1c02e49d.png)

Reply:
done!
<img width="888" alt="image" src="https://user-images.githubusercontent.com/32746338/123716862-77827400-d830-11eb-82dc-302a60e397db.png">